docker/build.sh

```diff
@@ -18,6 +18,12 @@ if ! docker images hyperledger/iroha-dev | grep -q hyperledger/iroha-dev; then
   fi
 fi
 
+# Overwrite docker image
+if [ "$1" = "-f" ]; then
+  echo "yes read -f option overlap new iroha.tar"
```

Comment:
Executing `clear.sh` removes `iroha.tar` and docker process and image used `iroha-docker`.
I thinked `build.sh` don't overwrite the above file. Is it wrong?